dlt/_workspace/helpers/dashboard/dlt_dashboard.py

```diff
@@ -72,20 +72,32 @@ def home(
     else:
         _buttons: List[Any] = []
         _buttons.append(dlt_refresh_button)
+        _pipeline_execution_summary: mo.Html = None
+        _last_load_packages_info: mo.Html = None
         if dlt_pipeline:
             _buttons.append(
                 mo.ui.button(
                     label="<small>Open pipeline working dir</small>",
                     on_click=lambda _: utils.open_local_folder(dlt_pipeline.working_dir),
                 )
             )
-        if dlt_pipeline and (local_dir := utils.get_local_data_path(dlt_pipeline)):
-            _buttons.append(
-                mo.ui.button(
-                    label="<small>Open local data location</small>",
-                    on_click=lambda _: utils.open_local_folder(local_dir),
+            if local_dir := utils.get_local_data_path(dlt_pipeline):
+                _buttons.append(
+                    mo.ui.button(
+                        label="<small>Open local data location</small>",
+                        on_click=lambda _: utils.open_local_folder(local_dir),
+                    )
+                )
+            if dlt_pipeline.last_trace:
```

Comment:
last_trace loads trace and is pretty heavy. better use := to get trace object in `if`